packages/webapp/src/webview/ui/components/FeedbackSection/FeedbackSection.tsx

```diff
@@ -22,6 +22,7 @@ export function FeedbackSection(): ReactElement {
             <h3>{i18next.t('PLEASE_TELL_US_IF_THIS_ANSWER_WAS_HELPFUL')}</h3>
             <FocusZone direction={FocusZoneDirection.horizontal} className="feedback-subcontainer" role="tree">
                 <button
+                    data-testid="solved-issue-button"
```

Comment:
**Proposal**: Instead of setting the `data-testid` can we just set `id` and configure react testing library to use `id` as `testIdAttribute`. Configuration can be done with
```typescript
configure({ testIdAttribute: 'id' })
```
(see also https://testing-library.com/docs/dom-testing-library/api-configuration/)

This way you still can use 

```typescript
screen.getByTestId('solved-issue-button');
```
but also CSS or other productive code can safely use the id to access the element.

I would do this change for occurrences: `data-testid` -> `id`

Reply:
Sure, no problem. Done